.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+
+	// Lets you preview .dmi files in the editor
```

Comment:
Yes, GitHub says this is invalid. However, this is valid in the 'JSON with comments' format used by VSCode.